CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.2.3
+
+- Fix #90: Apply locale based messaging
```

Comment:
Probably you would like to change `Fix` to `Issue` or vice versa to have consistency. Also, no need of colon.